src/Fields/constant_field.jl

```diff
@@ -21,6 +21,6 @@ ConstantField(constant) = ConstantField{3}(constant)
 
 const CF = Union{ConstantField, ZeroField, OneField}
 
-@inline Base.summary(f::CF) = string("ConstantField(", prettysummary(f.constant), ")")
+@inline Base.summary(f::CF) = string("ConstantField(", f[1, 1, 1], ")")
```

Comment:
Why don't we use `prettysummary` here?

Reply:
It was not the actual constant, it was approximated

Reply:
I still think its better to use `prettysummary` for `summary`, which is consistent with julia Base (ie we use `Base.Ryu.writeshortest` which is also used by julia `Base` for summaries of floats).

Reply:
I propose we just keep using `prettysummary` everywhere, and then change the behavior of `prettysummary` if we want to do that. That way we don't have to make tiny changes all throughout the code.

Reply:
In a different PR. Let's change less in this PR.